### PR TITLE
ENG-2121: Use dark logo for Aqueduct engine.

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
This PR replaces the light logo being used in the engine column with the dark logo.

## Related issue number (if any)
ENG-2121

## Loom demo (if any)
Screenshot:
image

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/utils/integrations.ts [hunk 1]
```diff
@@ -253,7 +253,7 @@ export const IntegrationCategories = {
 };
 
 export const ServiceLogos: ServiceLogo = {
-  ['Aqueduct']: `${logoBucket}/aqueduct-logo-light/1x/logo_light_blue.png`,
+  ['Aqueduct']: `${logoBucket}/aqueduct-logo-dark/small/2x/aqueduct-logo-dark-small%402x.png`,
   ['Postgres']: `${integrationLogosBucket}/440px-Postgresql_elephant.svg.png`,
```
Comment (agiron123) [inline src/ui/common/src/utils/integrations.ts:257]: Future task, but we should better standardize the path names in our S3 buckets.

**Comment by vsreekanti** [comment]
Hm, this is a little darker than I expected. Can we try what it would look like to use the light top/dark bottom one?

**Comment by agiron123** [comment]
@vsreekanti Here's a screenshot of the two tone variant:
image

I think it looks a little brighter

**Comment by vsreekanti** [comment]
I kinda like this one more. What do you think?

**Comment by vsreekanti** [comment]
Great, let's go with this one!

**Review by vsreekanti — APPROVED** [review]

**Pull request merged**